lstchain/image/muon/muon_analysis.py

```diff
@@ -365,23 +362,21 @@ def radial_light_distribution(centre_x, centre_y, pixel_x, pixel_y, image):
     standard_dev, skewness
     """
     # Convert everything to degrees:
-    x0 = centre_x.to(u.deg).value
-    y0 = centre_y.to(u.deg).value
-    pix_x = pixel_x.to(u.deg).value
-    pix_y = pixel_y.to(u.deg).value
-
-    pix_r = np.hypot(pix_x-x0, pix_y-y0)
+    x0 = center_x.to_value(u.deg)
+    y0 = center_y.to_value(u.deg)
+    pix_x = pixel_x.to_value(u.deg)
+    pix_y = pixel_y.to_value(u.deg)
 
+    pix_r = np.sqrt((pix_x-x0)**2 + (pix_y-y0)**2)
+    
     # mean, standard deviation & skewness of light distribution along ring radius.
     # ring_radius calculated elsewhere is approximately equal to "mean", but not exactly, so we recalculate it here:
-    mean = np.sum(np.multiply(image,pix_r))/np.sum(image)
-    standard_dev = np.sqrt(np.sum(np.multiply(image,np.square(pix_r-mean)))/np.sum(image))
-    skewness =  np.sum(np.multiply(image,np.power((pix_r-mean)/standard_dev,3.)))/np.sum(image)
-    excess_kurtosis = np.sum(np.multiply(image,np.power((pix_r-mean)/standard_dev,4.)))/np.sum(image) - 3.
+    mean = np.average(pix_r, weights=image)
+    delta_r = pix_r - mean
+    standard_dev = np.sqrt(np.average(delta_r**2, weights=image))
+    skewness = np.average((delta_r/standard_dev)**3, weights=image)
```

Comment:
Dividing by standard dev after averaging will divide a number by a number instead of an array by a number, improving speed

Reply:
Indeed, done.